packages/front-end/components/Features/RuleModal/index.tsx

```diff
@@ -443,12 +443,17 @@ export default function RuleModal({
           );
         }
 
-        track("Create Experiment", {
-          source: "experiment-ref-new-rule-modal",
-          numTags: feature.tags?.length || 0,
-          numMetrics: 0,
-          numVariations: values.values.length || 0,
-        });
+        track(
+          values.experimentType === "multi-armed-bandit"
+            ? "CreateBandit"
```

Comment:
```suggestion
            ? "Create Bandit"
```

Reply:
ok, so long as you fix it in the create modal as well